coordinator/src/step_2.rs

```diff
@@ -38,7 +38,7 @@ fn request_message(
     logger: &mut dyn Write,
     commitments: BTreeMap<Identifier, SigningCommitments>,
 ) -> Result<SigningPackage, Box<dyn std::error::Error>> {
-    let message = if args.message == "-" {
+    let message = if args.message.is_empty() {
```

Comment:
This will require changing the documentation of `message` inside the `Args` struct (which is used by `clap` when you call the binary with `-h`) to reflect this.

(I'm not sure why I picked "-", it seems easier to use empty)

Reply:
Yes, ok. The tests were failing on "-" as it was being read as empty. I'm not entirely sure why but I saw no issue in just changing it. I'll update the documentation.